contracts/LiquidityBridgeContract.sol

```diff
@@ -418,6 +461,12 @@ contract LiquidityBridgeContract {
 
         return keccak256(encodeQuote(quote));
     }
+
+    function validateAndHashPegOutQuote(PegOutQuote memory quote) private view returns (bytes32) {
+        require(address(this) == quote.lbcAddress, "Wrong LBC address");
+
```

Comment:
shouldn't we also include here this require, as in validateAndHashQuote?
`require(address(bridge) != quote.contractAddress, "Bridge is not an accepted contract address");`